code/modules/antagonists/heretic/knowledge/side_knock_flesh.dm

```diff
@@ -0,0 +1,27 @@
+// Sidepaths for knowledge between Knock and Flesh.
+
+/datum/heretic_knowledge/spell/apetra_vulnera
+	name = "Apetra Vulnera"
+	desc = "Grants you Apetra Vulnera, a spell \
+		that causes heavy bleeding on all bodyparts of the victim that have more than 15 brute."
+	gain_text = "Flesh opens, and blood spills. My master seeks sacrifice, and I shall appease."
+	next_knowledge = list(
+	/datum/heretic_knowledge/spell/blood_siphon,
+	/datum/heretic_knowledge/void_cloak,
+	)
+	spell_to_add = /datum/action/cooldown/spell/pointed/apetra_vulnera
+	cost = 1
+	route = PATH_SIDE
+
+/datum/heretic_knowledge/spell/opening_blast
+	name = "Wave Of Desperation"
+	desc = "Grants you Wave Of Desparation, a spell that cannot be casted uncuffed, \
```

Comment:
'cannot be casted uncuffed'? That seems like a bad idea to me. Just make it weaker when uncuffed

Reply:
> it literally knocks down everyone adjacent and removes your bolas and uncuffs you AND throws everyone else into walls

Yes so make it weaker but not literally unusable otherwise. Otherwise theres a pretty good chance you won't even get an opportunity to use it if the crew is smart and metas not cuffing or bolaing you.

Reply:
then dont buy the knowledge